src/store/plugins/HostStatusPlugin.js

```diff
@@ -0,0 +1,37 @@
+const HostStatusPlugin = store => {
+  let ws;
+  const data = {
+    paths: ['/xyz/openbmc_project/state/host0'],
+    interfaces: ['xyz.openbmc_project.State.Host']
+  };
+
+  const initWebSocket = () => {
```

Comment:
A comment or 2 in here on *why* we are doing this could help when we upstream. Also maybe a link to the document that explains how all this works might also be helpful https://github.com/openbmc/docs/blob/b41aff0fabe137cdb0cfff584b5fe4a41c0c8e77/rest-api.md#event-subscription-protocol

Reply:
Great documentation..hadn't seen this earlier. I will plan to add comments